app/src/main/java/org/oppia/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -84,10 +99,46 @@ class AudioFragmentPresenter @Inject constructor(
       it.audioFragment = fragment as AudioFragment
       it.lifecycleOwner = fragment
     }
-
+    subscribeToProfileLiveData()
     return binding.root
   }
 
+  private val profileLiveData: LiveData<Profile> by lazy {
+    getProfileData()
+  }
+
+  private fun getProfileData(): LiveData<Profile> {
+    return Transformations.map(
+      profileManagementController.getProfile(profileId),
+      ::processGetProfileResult
+    )
+  }
+
+  private fun subscribeToProfileLiveData() {
+    profileLiveData.observe(activity, Observer<Profile> { result ->
+      audioLanguage = result.audioLanguage
+      languageSelected(getAudioLanguage(audioLanguage))
+    })
+  }
+
+  /** Gets language code by [AudioLanguage]. */
+  fun getAudioLanguage(audioLanguage: AudioLanguage): String {
+    return when (audioLanguage) {
+      AudioLanguage.NO_AUDIO -> "No Audio"
```

Comment:
>Will the No Audio value be processed by languageSelected correctly? Maybe we should create a sentinel string >value that represents an invalid language code and use that, instead.
@mschanteltc  in app default options of audio contains "No Audio"... Please suggest why we need this

Reply:
When the user opens up any exploration, it will always play the default audio. However some users may not want this because it takes up their data if they are streaming the exploration. In this case, "No Audio" is used so that when the user opens an exploration, the audio is turned off.

Reply:
@mschanteltc 
Does this mean that as soon as the exploration is opened and if the learner has "Default Audio" selected, the audio will start even without clicking on the play button?

1. If yes, then why do we need such functionality, because this is not how the web is doing right?
2. If no, then "No Audio" does not make sense to as if that is what learner wants (meaning does not want to play audio), then the learner won't click on the play button.

Reply:
Those are valid points, so with the first point in mind, let's remove "No Audio" from the list. When the user opens the Exploration Player, the audio will not play automatically, similar to how the web functionality is made. However if the user plays the audio and goes to the next page in an exploration, the audio will play automatically.

Reply:
@mschanteltc This makes more sense to me.

Reply:
> Those are valid points, so with the first point in mind, let's remove "No Audio" from the list. When the user opens the Exploration Player, the audio will not play automatically, similar to how the web functionality is made. However if the user plays the audio and goes to the next page in an exploration, the audio will play automatically.

Done.

Reply:
Edited the [PRD](https://docs.google.com/document/d/1Ae587Q1ggfoca0j5j44LO3VFx_Z35oklyTcPZbz2w3I/edit?ts=5d02d804#heading=h.rldyjsnjg1c2) to reflect these changes.